src/harness/reference_models/interference/aggregate_interference.py

```diff
@@ -0,0 +1,313 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
```

Comment:
In your latest change, you have reinjected the files in interference/ that were removed from the initial PR when you broke it in piece.

I guess we have another GIT issue here that will prevent clean merging. Not exactly sure how to clean it up without messing the PR. Will think about it.